analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent.razor

```diff
@@ -0,0 +1,98 @@
+﻿@page "/component"
+
+<h3>Component</h3>
+
+@code {
+    [Parameter]
+    [SupplyParameterFromQuery] @* Compliant *@
+    public bool Param { get; set; }
+
+    [Parameter]
+    public string SupplyParameterFromQueryAttributeMissing { get; set; } @* Compliant: missing [SupplyParameterFromQuery] *@
+
+    [SupplyParameterFromQuery]
+    public string ParameterAttributeMissing { get; set; } @* Compliant: missing [Parameter] *@    [Parameter]
```

Comment:
Not sure exactly what happened here but the the test cases in #8340 look a bit different: for example it's missing the attribute list decorated property:
```cs
[Parameter, SupplyParameterFromQuery]
public int MyInt { get; set; } // Compliant
```